textrepo-app/src/main/java/nl/knaw/huc/resources/rest/DocumentFilesResource.java

```diff
@@ -19,21 +22,24 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.Response;
 import java.util.UUID;
 
 import static java.util.Objects.requireNonNull;
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 import static nl.knaw.huc.helpers.Paginator.toResult;
 
-@Api(tags = {"documents"})
+@Api(tags = {"documents", "files"})
 @Path("/rest/documents/{docId}/files")
 public class DocumentFilesResource {
 
   private static final Logger log = LoggerFactory.getLogger(DocumentFilesResource.class);
   private final DocumentFilesService documentFilesService;
   private final Paginator paginator;
 
+  private static class ResultTextRepoFilePage extends ResultPage<ResultTextRepoFile> {}
```

Comment:
Mijn 'save actions / code formatter' plaatst een lege regel tussen de `{}` (lege implementatie).

Even gelijktrekken, hetzij hier in de code, hetzij in de code formatting settings die we gebruiken.

Reply:
Gefixt in 000f1bd